include/mbedtls/ctr_drbg.h

```diff
@@ -8,8 +8,11 @@
  * Recommendation for Random Number Generation Using Deterministic Random
  * Bit Generators</em>.
  *
- * The Mbed TLS implementation of CTR_DRBG uses AES-256 as the underlying
- * block cipher.
+ * The Mbed TLS implementation of CTR_DRBG uses AES-256 (default) or AES-128 
+ * as the underlying block cipher.
+ * 
```

Comment:
Don't add trailing whitespace in commit "enable the use of variable size keys in ctr_drbg (was hard-coded 256bit before)".